website/docs/r/spring_cloud_app.html.markdown

```diff
@@ -78,7 +78,11 @@ An `custom_persistent_disk` block exports the following:
 
 An `identity` block supports the following:
 
-* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Cosmos Account. The only possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Spring Cloud Application. Possible values are `SystemAssigned`, `UserAssigned`, `SystemAssigned, UserAssigned` (to enable both).
+
+* `identity_ids` - (Optional) A list of User Assigned Managed Identity IDs to be assigned to this Spring Cloud Application.
```

Comment:
A general comment, the list of resource IDs are obsolete now, see [Migrating from the deprecated "array" model to the "dictionary" model section](https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingUserAssigned.html).

Reply:
Same above, it simplifies how user inputs this property, it has internal logics to convert/handle the difference between tf schema and api model.

Reply:
Not sure if terraform will use only PUT http method or not.

With a raw patch request, customer can remove a user identity with a dictionary, see below:

## Exist identities
Suppose existed identities on app: userIdentity1,  userIdentity2
## Request
Then patch with below:
```json
userAssignedIdentities: {
  'resourceId1': null
}
```
## Result
Result: userIdentity2 is kept.

Reply:
Yes, tf will use PUT, so the identity ids can be updated.

Reply:
Thanks, great to know.